src/Commands.pm

```diff
@@ -1676,13 +1670,13 @@ sub cmdDebug {
 
 		message center(T(" Debug information "), 56, '-') ."\n".
 			TF("ConState: %s\t\tConnected: %s\n" .
-			"AI enabled: %s\t\tAI_forcedOff: %s\n" .
+			"AI enabled: %s\n" .
```

Comment:
Refactors on these multi-line translated messages causes the whole message to lose its translations. What do you think about making it policy to split them up into individual sentences (or maybe lines)? We don't have to do it here, I'm just thinking about it.